src/proc/temporal-filter.cpp

```diff
@@ -146,7 +145,6 @@ namespace librealsense
     {
         std::lock_guard<std::mutex> lock(_mutex);
         _alpha_param = val;
-        _one_minus_alpha = 1.f - _alpha_param;
```

Comment:
Not used?

Reply:
I removed it as a member and added a calculation in the one place it was used. This way no need to lock for `alpha` and `one_minus_alpha` used together.

Reply:
I see now that the corresponding change in `temporal-filter.h` was not committed properly. Will fix.

Reply:
What happens if the user change the alpha between lines 37-38 in the usage?
I see the alpha is set with a lock.
Can we have a use case when alphe is set locally / then changed and then we set _one_minus_alpha?

Maybe better to do (use the local value we set before):

```
float alpha = _alpha_param;
float one_minus_alpha = 1.f - alpha ;
```

Reply:
You are right, that was my intention to begin with. Fixed.